src/CodeGen_Hexagon.cpp

```diff
@@ -531,6 +534,8 @@ void CodeGen_Hexagon::init_module() {
 
         // Broadcasts
         { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatw), u32v1,  "splat.w", {u32} },
+        { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatb), u8v1,   "splat.b", {u8}  },   // v62 - Broadcasts
+        { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplath), u16v1,  "splat.h", {u16} },   // v62 - Broadcasts
```

Comment:
There are no tests in simd_op_check.cpp for splatb and splath. I know there were some problems with these instructions w.r.t to our LLVM tools. If you are seeing those problems still, I suggest you remove these two lines for now and we can plan to fix all splat issues in a few weeks time.